src/app/index.ts

```diff
@@ -19,21 +18,22 @@ import { upgradeAdapter } from "./adapter";
 require("./vendor.scss");
 require("./index.scss");
 
-import config from "./config.provider";
-import signin from "./signin";
+upgradeAdapter.upgradeNg1Provider("hotkeys"); // needed for pushToTalk
 
-// Components
-import footerComponent from "./footer";
-import feedComponent from "./feed";
-import chatComponent from "./chat";
-import roomComponent from "./room";
+/* Register providers for browser, this is mandatory. */
```

Comment:
This comment comes from the example of angular2-modal, it's not enabled in this branch because needs a full migration to Angular 2. In the next PR removing Angular 1, this should be fixed.